src/Compatibility/Core/src/Compatibility.csproj

```diff
@@ -16,7 +16,7 @@
   </PropertyGroup>
 
   <PropertyGroup Condition="$(TargetFramework.Contains('-windows')) == true ">
-    <NoWarn>$(NoWarn);CA1416;CS8305</NoWarn>
+    <NoWarn>$(NoWarn);CA1416;CS8305;IL2059</NoWarn>
```

Comment:
Can we do `IL2059` conditionally on Windows? It is a useful warning on iOS/Android.

Not a blocker, though.